_docs/_latest/configure/scanner/rules.md

```diff
@@ -356,7 +358,7 @@ rules:
 
 ```yaml
 rules:
-  - name: sample cloudsql rule to search for publicly exposed instances
+  - name: sample Cloud SQL rule to search for publicly exposed instances
```

Comment:
@joecheuk and @blueandgold can we ensure the sample configuration and default rules use this same naming format in its comments?

Reply:
@mirons-google, this is easy to update in the default rule file.  

But in the configuration file, it will require code change. Do you mean like below, where we break up `cloudsql` into 2 words, `cloud_sql`?

current
```
    scanners:
        - name: cloudsql_acl
          enabled: true
```

proposed
```
    scanners:
        - name: cloud_sql_acl
          enabled: true
```

Reply:
Let's make the change in the default rule and add a TODO for the conf file. Can you do that and get it submitted before we submit this PR?

Reply:
Opened PR #1695  to change in rule file.  Opened issue #1694 to change in config file.  The TODO seems to stick out too much in the conf file, so opening an issue to track this.

Reply:
Resolved.